docs/usage.md

```diff
@@ -11,30 +11,37 @@ have to wait a whilst it sets up in the background (for those curious or interes
 [pyodide](mesnage-org.github.io/pgfinder-gui/) a [Web Assembly language (WASM)](https://webassembly.org/) in the
 background to install PGFinder and all its Python dependencies). Once loaded the page should look like the image below.
 
-![Initial screenshot of PGFinder GUI](img/pgfinder-gui001.png)
+![image](https://github.com/Mesnage-Org/pgfinder/assets/70374280/9537c200-5b48-4d50-ac72-1c8b62d83909)
```

Comment:
With `image` we lose what is known as the "alt-text" on the image so when you hover over it the work "image" appears, whilst previously it would be "Initial screenshot of PGFinder GUI".

```suggestion
![Initial screenshot of PGFinder GUI](https://github.com/Mesnage-Org/pgfinder/assets/70374280/9537c200-5b48-4d50-ac72-1c8b62d83909)
```